celery/canvas.py

```diff
@@ -1448,6 +1448,14 @@ class group(Signature):
 
     @classmethod
     def from_dict(cls, d, app=None):
+        """Create a group signature from a dictionary of tasks.
```

Comment:
This is not a dictionary of tasks but a dictionary which represents a group signature. Within it, there is a list of tasks in the group.
Please find a way to rephrase.

Reply:
Fix pushed.